docs/1.getting-started/5.seo-meta.md

```diff
@@ -33,9 +33,9 @@ This method does not allow you to provide reactive data. We recommend to use `us
 
 Shortcuts are available to make configuration easier: `charset` and `viewport`. You can also provide any of the keys listed below in [Types](#types).
 
-## `useHead`
+## [`useHead`](/docs/api/composables/use-head)
```

Comment:
```suggestion
## `useHead`
```

Reply:
I've now fixed all the links that appear in the title.